verl/trainer/config/ppo_trainer.yaml

```diff
@@ -925,6 +925,9 @@ trainer:
   # Device to run training on (e.g., "cuda", "cpu")
   device: cuda
 
+  # whether to use legacy role implementation
+  use_legacy_worker_impl: True
```

Comment:
can we default it to null or "auto", so that we can automatically switch to the new impl if user do not set this config